app/test/e2e/files/points_response_v1.json

```diff
@@ -7,8 +7,8 @@
         {
           "type": "Feature",
           "properties": {
-            "lon": "-69.76318359375",
```

Comment:
Any change the new implementation can return strings, as it did before? This conversion from string to number is technically a BC break